tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -1949,3 +2003,11 @@ def test_custom_measurement_size_filtering(self):
     @pytest.mark.xfail(reason="Having not supported")
     def test_having_condition(self):
         super().test_having_condition()
+
+    @pytest.mark.xfail(reason="wildcard not supported")
+    def test_transaction_wildcard(self):
+        super().test_having_condition()
+
+    @pytest.mark.xfail(reason="wildcard not supported")
+    def test_transaction_status_wildcard(self):
+        super().test_having_condition()
```

Comment:
I think you meant to reference the corresponding test in these two?

```suggestion
    @pytest.mark.xfail(reason="wildcard not supported")
    def test_transaction_wildcard(self):
        super().test_transaction_wildcard()

    @pytest.mark.xfail(reason="wildcard not supported")
    def test_transaction_status_wildcard(self):
        super().test_transaction_status_wildcard()
```

Reply:
Nah needed to delete these, ty